app/src/routes/project/WebviewArea/ResizeHandles.tsx

```diff
@@ -19,6 +19,17 @@ const ResizeHandles = observer(({ webviewSize, setWebviewSize }: ResizeHandlePro
     const editorEngine = useEditorEngine();
     const resizeHandleRef = useRef(null);
 
+    const handleMouseMove = (e: React.MouseEvent) => {
```

Comment:
@Sachinsingh2002 this is close but I don't think it's right. The change on 
[app/src/routes/project/LayersPanel/index.tsx](https://github.com/onlook-dev/studio/pull/161/files#diff-f0245d5cb38ab89312972b0120a231f7e191c7938935b2c5dd18f4c3cddbfe08) is correct. 

It would be better to add a resize handle (just a div with some event handlers) on the layers panel that captures mouse down event and does the resizing like you did here. 

This is what this component is referring to (the white handle).  
<img width="134" alt="Screenshot 2024-08-15 at 1 25 15 PM" src="https://github.com/user-attachments/assets/bea62978-81e2-4a8e-8e49-20f19635ca2f">